cmd/git_utils.go

```diff
@@ -159,7 +159,18 @@ func RunGitConfigLocalRemoteOriginURL(upstream string, dryrun bool) (string, err
 
 	upstreamStart := strings.Split(upstream, "/")[0]
 	kargs := []string{"config", "--local", "remote." + upstreamStart + ".url"}
-	return RunGit(kargs, dryrun)
+	remote, err := RunGit(kargs, dryrun)
+	if err != nil {
+		return remote, err
+	}
+
+	// Convert ssh remote to https
+	if strings.Contains(remote, "git@") {
+		remote = strings.Replace(remote, ":", "/", -1)
+		remote = strings.Replace(remote, "git@", "https://", -1)
```

Comment:
Good spot, I replaced the `.git`  with `/` since e.g `https://github.com/appsody/appsody/` works, and that means `https://github.com/appsody/appsody/tree/master` also works.
Have tested these locally